<launch>
  <param name="calibration_path" value="$(find rpl)/config/camera.xml" />
  <param name="file_path" value="$(find rpl)/config/source_position/"/>
  <param name="slop_ms" type="double"    value="1000.0"/>
  <param name="matrix_path" value="$(find rpl)/config/log_image_robot/Pose_2020-12-14_134019/"/>

  <node name="camera" pkg="rpl" type="camera.py" output="screen"/>
  <node name="ar" pkg="rpl" type="ar_detector.py" output="screen"/>
  <node name="estimate_ar_position" pkg="rpl" type="estimate_ar_position.py" output="screen"/>
  <node name="estimate_rb_position" pkg="rpl" type="estimate_rb_position.py" output="screen"/>

  <node name="pose_integrator" pkg="rpl" type="pose_integrator.py" output="screen"/>
  <node name="pose_confusion" pkg="rpl"  type="pose_confusion.py" output="screen"/>

  <node name="demo" pkg="rpl"  type="demo.py" output="screen"/>
</launch>
